frontend/src/AlertConfiguration/AlertConfigurationComponents/ConfigList.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, Fragment } from 'react';
```

Comment:
again, this file seems to look very similar to `ConfigForm.js` and `ConfigCard.js`...

was wondering about the reason behind this.. 

thanks!